buildpack/telemetry/telegraf.py

```diff
@@ -186,6 +188,7 @@ def update_config(m2ee, app_name):
         micrometer_metrics=metrics.micrometer_metrics_enabled(runtime_version),
         cf_instance_index=_get_app_index(),
         app_name=app_name,
+        appdynamics_enabled=appdynamics.machine_agent_enabled(),
```

Comment:
line 66 has appd_enabled, line 191 has appdynamics_enabled. Consistency please :)

Reply:
Removed from 66. 
For 191 added comment that for Telegraf config only Machine Agent makes sense